config/install/advancedqueue.advancedqueue_queue.os2web_audit.yml

```diff
@@ -0,0 +1,10 @@
+status: true
+dependencies: {  }
+id: os2web_audit
+label: os2web_audit
+backend: database
+backend_configuration:
+  lease_time: 300
+processor: cron
```

Comment:
This should be change to run via advancedqueue:queue:proces, so we can stop this queue if there is issue with the remote logger server.

Reply:
Fixed in 1bf3fcb8f9b17c5da64336d3d35492cec50db2ed.